canonical-web-and-design/maas-ui
# feat(legacy): display release notifications on the dashboard

## Done

- Add angular support for release notifications and display them on the dashboard.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a release notification (instructions here: https://github.com/canonical-web-and-design/MAAS-squad/issues/2065).
- Visit the dashboard. You should see the release notification.
- Visit another legacy page, you should not see the notification.


## Fixes

Fixes: #1601.
